### PR TITLE
[#454] Pin layout screen as that it android

### LINKED ISSUE #454: Create the pin layout page as that in Android
* **I'm submitting a ...**
  - [ ] bug report
  - [x] feature request

* **What is the current status?**
- This feature is not available in the Desktop app

* **What is the expected behavior?**
- Make this page so that the user can view the device pin layout 

* **Please tell us about your environment:**
  - OS with version number: Ubuntu 18.04

* **Would you like to work on it?**
Yes

### PR DESCRIPTION
**Pull request by AakashMallik**
Fixes #454 

* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
- [ ] Bug fix
- [x] Feature implementation
- [ ] Doc updates

* **What changes have you introduced?**
- Added a pin layout screen accessible from the home screen.
- Added pin guide tooltips for front-layout

* **Does this PR introduce a breaking change?**
No

* **Preview / Steps to verify your work**:

### CODE REVIEW

**Comment by CloudyPadmal** [comment]
In Android app, if user clicks on a pin, it will open an info dialog with a small description on that pin

**Comment by AakashMallik** [comment]
> In Android app, if user clicks on a pin, it will open an info dialog with a small description on that pin

Oh yes, I remember Neil told me about that. Has that feature been published? I don't see it working on my app.

**Comment by CloudyPadmal** [comment]
Screenshot_1564486019

**Comment by AakashMallik** [comment]
> Has to click on the pin.
> 
> 

Nice, will get this done ASAP.

**Comment by AakashMallik** [comment]
@CloudyPadmal The process of doing this was very time consuming. I have done the font-layout in this PR. WIll do backlayout in subsequent PR. LOC changes in this PR are > 300. I hope that is okay

File: src/screen/Layout/FrontLayout.js [hunk 1]
```diff
@@ -0,0 +1,858 @@
...
+          }
+          placement="top"
+        >
+          <Marker top={32} left={710}></Marker>
+        </Tooltip>
+        <Tooltip
+          title={
+            <React.Fragment>
+              <div style={{ fontSize: '16px' }}>
+                Progrmmable current source (3.3 mA)
```
Comment (CloudyPadmal) [inline src/screen/Layout/FrontLayout.js]: Programmable

File: src/screen/Layout/FrontLayout.js [hunk 2]
```diff
@@ -0,0 +1,858 @@
...
+          }
+          placement="top"
+        >
+          <Marker top={32} left={430}></Marker>
+        </Tooltip>
+        <Tooltip
+          title={
+            <React.Fragment>
+              <div style={{ fontSize: '16px' }}>
+                Voltage supply pin ( 3.3V )
```
Comment (CloudyPadmal) [inline src/screen/Layout/FrontLayout.js]: 3.3 V

File: src/screen/Layout/FrontLayout.js [hunk 3]
```diff
@@ -0,0 +1,858 @@
...
+          }
+          placement="top"
+        >
+          <Marker top={32} left={575}></Marker>
+        </Tooltip>
+        <Tooltip
+          title={
+            <React.Fragment>
+              <div style={{ fontSize: '16px' }}>
+                Voltage supply pin ( 3.3V )
```
Comment (CloudyPadmal) [inline src/screen/Layout/FrontLayout.js]: 3.3 V

File: src/screen/Layout/FrontLayout.js [hunk 4]
```diff
@@ -0,0 +1,858 @@
...
+          }
+          placement="top"
+        >
+          <Marker top={32} left={822}></Marker>
+        </Tooltip>
+        <Tooltip
+          title={
+            <React.Fragment>
+              <div style={{ fontSize: '16px' }}>
+                Programmable voltage source 2 (-+ 3.3 V)
```
Comment (CloudyPadmal) [inline src/screen/Layout/FrontLayout.js]: Instead of -+, use https://graphemica.com/%C2%B1

File: src/screen/Layout/FrontLayout.js [hunk 5]
```diff
@@ -0,0 +1,858 @@
...
+          }
+          placement="top"
+        >
+          <Marker top={32} left={877}></Marker>
+        </Tooltip>
+        <Tooltip
+          title={
+            <React.Fragment>
+              <div style={{ fontSize: '16px' }}>
+                Programmable voltage source 1 (-+ 5 V)
```
Comment (CloudyPadmal) [inline src/screen/Layout/FrontLayout.js]: -+

File: src/screen/Layout/FrontLayout.js [hunk 6]
```diff
@@ -0,0 +1,858 @@
...
+          }
+          placement="bottom"
+        >
+          <Marker top={568} left={343}></Marker>
+        </Tooltip>
+        <Tooltip
+          title={
+            <React.Fragment>
+              <div style={{ fontSize: '16px' }}>
+                Reciever pin for UART communication
```
Comment (CloudyPadmal) [inline src/screen/Layout/FrontLayout.js]: Receiver

File: src/screen/Layout/FrontLayout.js [hunk 7]
```diff
@@ -0,0 +1,858 @@
...
+            </React.Fragment>
+          }
+          placement="bottom"
+        >
+          <Marker top={568} left={658}></Marker>
+        </Tooltip>
+        <Tooltip
+          title={
+            <React.Fragment>
+              <div style={{ fontSize: '16px' }}>PWM Generator pin 1</div>
```
Comment (CloudyPadmal) [inline src/screen/Layout/FrontLayout.js]: generator

File: src/screen/Layout/FrontLayout.js [hunk 8]
```diff
@@ -0,0 +1,858 @@
...
+            </React.Fragment>
+          }
+          placement="bottom"
+        >
+          <Marker top={568} left={714}></Marker>
+        </Tooltip>
+        <Tooltip
+          title={
+            <React.Fragment>
+              <div style={{ fontSize: '16px' }}>PWM Generator pin 2</div>
```
Comment (CloudyPadmal) [inline src/screen/Layout/FrontLayout.js]: generator

File: src/screen/Layout/FrontLayout.js [hunk 9]
```diff
@@ -0,0 +1,858 @@
...
+            </React.Fragment>
+          }
+          placement="bottom"
+        >
+          <Marker top={568} left={771}></Marker>
+        </Tooltip>
+        <Tooltip
+          title={
+            <React.Fragment>
+              <div style={{ fontSize: '16px' }}>PWM Generator pin 3</div>
```
Comment (CloudyPadmal) [inline src/screen/Layout/FrontLayout.js]: generator

File: src/screen/Layout/FrontLayout.js [hunk 10]
```diff
@@ -0,0 +1,858 @@
...
+            </React.Fragment>
+          }
+          placement="bottom"
+        >
+          <Marker top={568} left={825}></Marker>
+        </Tooltip>
+        <Tooltip
+          title={
+            <React.Fragment>
+              <div style={{ fontSize: '16px' }}>PWM Generator pin 4</div>
```
Comment (CloudyPadmal) [inline src/screen/Layout/FrontLayout.js]: generator

File: src/screen/Layout/FrontLayout.js [hunk 11]
```diff
@@ -0,0 +1,858 @@
...
+          }
+          placement="right"
+        >
+          <Marker top={264} left={1075}></Marker>
+        </Tooltip>
+        <Tooltip
+          title={
+            <React.Fragment>
+              <div style={{ fontSize: '16px' }}>
+                Oscilloscope channel input 3
```
Comment (CloudyPadmal) [inline src/screen/Layout/FrontLayout.js]: channel 3 input

File: src/screen/Layout/FrontLayout.js [hunk 12]
```diff
@@ -0,0 +1,858 @@
...
+          }
+          placement="right"
+        >
+          <Marker top={293} left={1075}></Marker>
+        </Tooltip>
+        <Tooltip
+          title={
+            <React.Fragment>
+              <div style={{ fontSize: '16px' }}>
+                Oscilloscope channel input 2
```
Comment (CloudyPadmal) [inline src/screen/Layout/FrontLayout.js]: channel 2 input

File: src/screen/Layout/FrontLayout.js [hunk 13]
```diff
@@ -0,0 +1,858 @@
...
+          }
+          placement="right"
+        >
+          <Marker top={319} left={1075}></Marker>
+        </Tooltip>
+        <Tooltip
+          title={
+            <React.Fragment>
+              <div style={{ fontSize: '16px' }}>
+                Oscilloscope channel input 1
```
Comment (CloudyPadmal) [inline src/screen/Layout/FrontLayout.js]: channel 1 input

**Comment by AakashMallik** [comment]
@CloudyPadmal Changes addressed. Thank you for your detailed review, sir.

**Pull request merged**